pkg/function/server/server.go

```diff
@@ -105,5 +125,10 @@ func (s *server) Start(ctx context.Context, inputOptions ...Option) {
 	<-ctxWithSignal.Done()
 	log.Println("Got a signal: terminating gRPC server...")
 	defer log.Println("Successfully stopped the gRPC server")
-	grpcSvr.GracefulStop()
+	s.grpcServer.GracefulStop()
+}
+
+func (s *server) Stop() {
```

Comment:
Why do we need this?

Reply:
I added this `Stop` function such that in our unit test, we can explicitly call this method to stop the grpc server after we finish sending all out test requests. (to address comment: https://github.com/numaproj/numaflow-go/pull/31#discussion_r1067655652)

Reply:
What you need to do is using a cancellable `ctx` for `Start()`, and then `cancel()` that ctx will shut down the server.